src/SDKs/ServiceBus/data-plane/Directory.Build.props

```diff
@@ -9,13 +9,7 @@
 
     <!-- Do not inherit implicit dependencies from the engineering system during build or packaging -->
     <ImportDefaultReferences>false</ImportDefaultReferences>
-
-    <!--
-      Override the inherited behavior of forcing test assemblies to be unsigned; this is needed
-      due to the use of "InternalsVisibleTo" grants for test visibility of non-public members
-    -->
-    <SignTestAssembly>true</SignTestAssembly>
```

Comment:
Thanks for cleaning this up I missed it because I only did a search in *.csproj :)